source/common/runtime/runtime_features.cc

```diff
@@ -116,6 +116,8 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_refresh_rtt_after_request);
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_quic_reject_all);
 // TODO(adisuissa): enable by default once this is tested in prod.
 FALSE_RUNTIME_GUARD(envoy_restart_features_use_eds_cache_for_ads);
+// TODO(yanavlasov) change to true when UHV is sufficently tested
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_enable_header_validator);
```

Comment:
so akin to balsa and oghttp2 I think we want a master tracking issue with a summary at the top of what this is and why we're doing it.    We can link that here as well.

Reply:
Done